repl/scala-2.11/src/main/scala/org/apache/spark/repl/Main.scala

```diff
@@ -44,7 +44,9 @@ object Main extends Logging {
 
   private def scalaOptionError(msg: String): Unit = {
     hasErrors = true
+    //scalastyle:off
```

Comment:
Yeah revert these. I am not sure we need to check or modify this file because it's a clone of Scala's code